php/php.editor/src/org/netbeans/modules/php/editor/parser/astnodes/CastExpression.java

```diff
@@ -32,7 +32,8 @@ public enum Type {
     	ARRAY, // 'array'
     	OBJECT, // 'object'
     	BOOL, // 'bool'
-    	UNSET// 'unset'
+    	UNSET,// 'unset'
+        FLOAT, // 'float'
```

Comment:
We should not only add `FLOAT` but also deprecate `REAL`, no? Ideally with a comment that it will be removed in the future, to follow PHP type system.

Reply:
Agreed. 

e.g.
```java
        /**
         * @deprecated will be removed in the future, to follow PHP type system,
         * use {@link Type#FLOAT} instead
         */
        REAL, // 'real'
```